src/components/header.tsx

```diff
@@ -1,109 +1,237 @@
+          </ListItemButton>
+        </ListItem>
+        <ListItem disablePadding>
+          <ListItemButton
+            component="a"
+            href="https://advisorysg.typeform.com/to/HQ8nWq2r#source=mentorsite"
             target="_blank"
             rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/1aA3l_WQ1ehGO8UBKuRFLqj4URpQbAVzN/view?usp=sharing"
           >
-            here
-          </a>{" "}
-          to view a sample application response.
-        </p>
-        <p>
-          Click{" "}
-          <a
-            target="_blank"
-            rel="noopener noreferrer"
-            href="https://drive.google.com/file/d/19lUr2BchbYFAPMjEbN_M2mf1pyw-n7pj/view?usp=sharing"
+            <ListItemText primary="Apply Now" />
+          </ListItemButton>
+        </ListItem>
+      </List>
+      <Divider />
+    </div>
+  );
+  const container =
+    window !== undefined ? () => window().document.body : undefined;
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <AppBar style={{ background: "#f89b27" }} className="toolbar-component">
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            sx={{ mr: 2, display: { sm: "none" } }}
           >
-            here
-          </a>{" "}
-          to view a list of 48 industries to filter mentors by industry.
-        </p>
-      </div>
+            <MenuIcon />
+          </IconButton>
+          <img
+            className="nav-logo"
+            src={advisoryLogo}
+            alt="Advisory"
+            style={{ alignSelf: "center" }}
+          />
+          <a className="nav-back-to-main-site" href="https://advisory.sg">
+            &#x3c; back to main site
+          </a>
+        </Toolbar>
+      </AppBar>
+      <Box
+        component="nav"
+        sx={{ width: { sm: drawerWidth }, flexShrink: { sm: 0 } }}
+      >
+        <Drawer
+          container={container}
+          variant="temporary"
+          open={mobileOpen}
+          onClose={handleDrawerToggle}
+          ModalProps={{
+            keepMounted: false, // Better open performance on mobile.
+          }}
+          sx={{
+            display: { xs: "block", sm: "none" },
+            "& .MuiDrawer-paper": {
+              boxSizing: "border-box",
+              width: drawerWidth,
+            },
+          }}
+        >
+          {drawer}
+        </Drawer>
+      </Box>
+    </Box>
+  );
+}
+//const aboutus = document.getElementById("aboutus");
+const Header = () => (
+  <div className="header">
+    <ResponsiveDrawer />
+    <div className="content-body">
+      <Box component="main">
+        <div className="header-bottom">
+          <img
+            className="header-mentorship-logo"
+            src={advisoryMentorshipLogo}
+            alt="Advisory Mentorship Programme"
+          />
+          <img
+            id="sponsors"
```

Comment:
This `id` should be changed to `partners`, not sure why it was `sponsors`. Do change as indicated above.